docs/api/index.md

```diff
@@ -244,6 +245,62 @@ test('props', () => {
 })
 ```
 
+### vModel
+
+Similar to `props`, but values passed to `vModel` will be updated when using the [`update:modelValue` syntax](https://v3.vuejs.org/guide/component-custom-events.html#v-model-arguments).
+
+**Signature:**
+
+```ts
+vModel?: (RawProps & Props) | ({} extends Props ? null : never)
+```
+
+**Details:**
+
+`Component.vue`:
+
+```vue
+<template>
+  <input @input="updateValue" />
+  {{ color }}
+</template>
+
+<script>
+export default {
+  props: {
+    color: String
+  },
+  methods: {
+    handle($event) {
+      this.$emit('update:color', $event.target.value)
+    }
+  }
+}
+</script>
+```
+
+`Component.spec.js`:
+
+```js
+it('updates v-model automatically', async () => {
```

Comment:
Not that important, but examples in docs use `test` instead of `it` :) #nitpicking

```suggestion
test('updates v-model automatically', async () => {
```